hera_cal/io.py

```diff
@@ -7,26 +7,537 @@
 import operator
 import os
 import copy
-import gc as garbage_collector
+import warnings
 from functools import reduce
+from hera_cal.utils import polnum2str, polstr2num, jnum2str, jstr2num
+from hera_cal.utils import split_pol, conj_pol
+import collections
 
-polstr2num = {'I': 1, 'Q': 2, 'U': 3, 'V': 4, 'RR': -1, 'LL': -2, 'RL': -3, 'LR': -4, 'xx': -5, 'yy': -6, 'xy': -7, 'yx': -8}
-polnum2str = {val: key for key, val in polstr2num.items()}
 
-jonesnum2str = {-5: 'x', -6: 'y'}
-jonesstr2num = {'x': -5, 'y': -6}
+class HERACal(UVCal):
```

Comment:
Although it falls in line given the naming convention of HERAData, I'm not a fan of the name `HERACal` given its similarity with the name of the software package. But I don't really care that much, so if you want to keep it I'm fine w/ it

Reply:
I get that, though I haven't heard the objection from anyone else so it might just be a matter of taste. If I hear more complaints, it's not too hard to change.